ui/lib/core/addon/components/shamir/dr-token-flow.hbs

```diff
@@ -0,0 +1,136 @@
+            as |cmd|
+          }}
+            <HoverCopyButton @copyValue={{cmd}} />
+            <code class="is-word-break">{{cmd}}</code>
+          {{/let}}
+          {{! template-lint-enable quotes }}
+        </div>
+      </div>
+    </div>
+    <div>
+      <button type="button" class="button is-primary" {{on "click" this.onCancelClose}}>
+        Clear &amp; Close
+      </button>
+    </div>
+  {{else if this.started}}
+    <Shamir::Form
+      @action={{@action}}
+      @progress={{this.progress}}
+      @threshold={{this.threshold}}
+      @errors={{this.errors}}
+      @inputLabel="Root key portion"
+      @buttonText="Generate token"
+      @onSubmit={{this.onSubmitKey}}
+      @otp={{this.otp}}
+      @alwaysShowProgress={{true}}
+      data-test-dr-token-flow-step="shamir"
+    >
+      <p>Generate an operation token by entering a portion of the
+        <strong>primary's root key</strong>. Once all portions are entered, the generated token may be used to manage your
+        secondary Disaster Recovery cluster.
+      </p>
+    </Shamir::Form>
+  {{else if this.generateWithPGP}}
+    <ChoosePgpKeyForm
+      @onCancel={{fn (mut this.generateWithPGP) false}}
+      @onSubmit={{this.usePgpKey}}
+      @formText={{this.pgpText.form}}
+      @confirmText={{this.pgpText.confirm}}
+      @buttonText="Generate operation token"
+      data-test-dr-token-flow-step="choose-pgp"
+    />
+  {{else}}
+    {{! Not started }}
+    <form
+      {{on "submit" this.startGenerate}}
+      id="shamir"
+      aria-label="shamir generate form"
+      data-test-dr-token-flow-step="begin"
+    >
+      <MessageError @errors={{this.errors}} />
+      <div class="has-bottom-margin-m" data-test-shamir-modal-body>
+        <p>
+          Updating or promoting this cluster requires an operation token, generated by inputting the root key shares. If
+          you'd like to first encrypt the token with a PGP Key, click "Encrypt with PGP key" below, otherwise we can begin
+          generation of the operation token.
+        </p>
+      </div>
+      <div class="field is-grouped is-flex-center">
+        <div class="control is-flex-row">
+          <button type="button" class="link" {{on "click" (fn (mut this.generateWithPGP) true)}} data-test-use-pgp-key-cta>
+            Provide PGP Key
+          </button>
+        </div>
+        <div class="control">
+          <span class="has-side-padding-s">
+            or
+          </span>
+        </div>
+        <div class="control">
+          <button type="submit" class="button is-primary" data-test-generate-token-cta>
+            Generate operation token
+          </button>
+        </div>
+      </div>
+    </form>
+  {{/if}}
+</section>
+<footer class="modal-card-foot modal-card-foot-outlined">
+  <button type="button" class="button is-secondary" {{on "click" this.onCancelClose}} data-test-shamir-modal-cancel-button>
+    {{if this.encodedToken "Close" "Cancel"}}
+  </button>
+</footer>
+{{! </Modal> }}
```

Comment:
Was the modal comment out here intentional?

Reply:
Good catch! Will remove